Automattic/wp-calypso
# Enable migration instructions feature for English locale

Related to #91815

## Proposed Changes

* Remove feature flag and enable the feature for English locale only ([See more](https://github.com/Automattic/wp-calypso/issues/91815#issuecomment-2234178532)).
* [Fix navigation to Migration Started step on `migration-signup` flow](https://github.com/Automattic/wp-calypso/pull/92766/commits/663e21e966152d607671c4d2a0a90770e7812feb).

## Why are these changes being made?


* The feature will be released, but we'll wait to remove the English check only when the translations are complete.

## Testing Instructions



* Make sure your locale is English in `https://wordpress.com/me/account`.
* Go to `http://calypso.localhost:3000/start`.
* Navigate through the flow until the migration instructions.
* Make sure you see the new migration instructions screen.
* Update your locale to a different one.
* Repeat the test, and make sure you see the old migration instructions screen.
* Update the locale to English again.
* Test the 2 other flows, and make sure they work properly:
  * `http://calypso.localhost:3000/setup/migration-signup`
  * `http://calypso.localhost:3000/setup/hosted-site-migration?from={FROM_URL}`

## Pre-merge Checklist
